Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM ubuntu:bionic as source
+
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y \
```

Comment:
Is this stage still necessary?

Reply:
> Is this stage still necessary?

No, removed it.
Added the apt-get upgrade so that the packages can be updated before commiting the image.

Reply:
Does anything of the stage get carried into the next one? You are only copying `.` and no of the commands should write there.

Reply:
> Does anything of the stage get carried into the next one? You are only copying `.` and no of the commands should write there.

Hmm, right.
As per my understanding, nothing is being carried over.
but I'm not sure.
@squaticaj can you please help here?

Reply:
In multi staged builds the layers are not included in the final image - see [here](https://docs.docker.com/develop/develop-images/multistage-build/) - only the one file that's copied by `COPY --from=<previousStage>`

Reply:
> In multi staged builds the layers are not included in the final image - see [here](https://docs.docker.com/develop/develop-images/multistage-build/) - only the one file that's copied by `COPY --from=<previousStage>`

Oh, that's nice.
I'll remove the apt-get upgrade then